cpf_segmentation/src/segmentation_test.cpp

```diff
@@ -58,7 +58,8 @@ int main(int argc, char** argv){
 
 
   /// Get pcd path from command line
-  std::string pcd_filename = argv[1];
+  std::string pcd_filename ="input/";
```

Comment:
spaces (codestyle)

Reply:
One more space is still skipped `std::string pcd_filename ="input/";` -> `std::string pcd_filename = "input/";`